mpmp/utilities/analysis_utilities.py

```diff
@@ -562,3 +562,36 @@ def process_coefs(coefs_df):
     return id_coefs
 
 
+def compare_all_data_types(results_df,
+                           sig_alpha,
+                           filter_genes=True,
+                           identifier='identifier',
+                           metric='aupr'):
+    """Run compare_results for each data type + assemble into dataframe"""
```

Comment:
I know you have several difference comparisons (i.e. your volcano plot and then your bar plots), maybe add a bit more detail here on what the plot is - "Compare the ability of each data type (i.e. gene expression, methylation) to predict if a gene is mutated or not. Return dataframe with AUPRC values per seed."

Reply:
Good idea, I got a bit lazy with this documentation. I'll add more detail.